packages/manager/modules/exchange/package.json

```diff
@@ -18,7 +18,6 @@
   "peerDependencies": {
     "@ovh-ux/ng-ovh-export-csv": "^1.0.0",
     "@ovh-ux/ng-ovh-http": "^4.0.1-beta.0",
-    "@ovh-ux/ng-ovh-otrs": "^9.0.1",
```

Comment:
This means we'll need to deprecate this module ?

Reply:
Yes :)